rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -71,7 +71,7 @@ object Runtime {
       BindPattern,
       InterpreterError,
       ListParWithRandom,
-      ListParWithRandomAndPhlos,
+      ListParWithRandom,
```

Comment:
Just double-checking. Do we need 2x `ListParWithRandom` here?

Reply:
Technically, we could probably delete one of the holes in here, and in RSpace. I'd wait with that though.

Reply:
No. Good catch.